js/ng.js

```diff
@@ -22,7 +22,10 @@ window.onload = () => {
         document.querySelectorAll(".addWitnessDiv").forEach(div => div.classList.remove("is-hidden"))
         document.querySelectorAll(".addWitnessBtn").forEach(btn => btn.classList.remove("is-hidden"))
         glossForm.querySelector(".dropGloss").classList.remove("is-hidden")
+       
+        
```

Comment:
It looks like you have only made whitespace changes in this file.  Can you `undiff` this and just checkout the whitespaces in this file as they are upstream from you